BUILD.bazel

```diff
@@ -0,0 +1,8 @@
+load("//:plugin.bzl", "CLOE_PLUGINS")
```

Comment:
not sure if i would keep the *.bzl on the top lvl (i have a feeling that in the long run there will be more *.bzl files in the long run, which might either lead to a cluttered top lvl or all over the repo distributes bzl files which is imho not so pleasant)

Reply:
Do you think it's acceptable to reorganize this for making it leave the experimental stage?

Reply:
That is: I want to do it later, when Bazel is more officially supported, and is not experimental.